README.md

```diff
@@ -1 +1,26 @@
-# virtual-pet
\ No newline at end of file
+Real pets are great, but they do have drawbacks. They smell. Your mum might be
```

Comment:
This is a good start but there are some important bits of info missing from this. Some things that would make it much better:
- installation instructions (ie cloning the repo and what commands are available),
- what technologies are required to use it, what the application dependencies are (`JavaScript`, `Jest` and `Node` in this case)
- a description of what the rules are for how the pet is fed, exercised, how long it will live, what are the max and min values for things like hunger, fitness etc.
- how the app is tested,
- what you would do if you had more time (if applicable)
- your name

The formatting could use a bit of attention too, [markdown live preview](https://markdownlivepreview.com/) is useful so you can play with what the file will look like without having to do multiple commits to the repo to see the results, have a look through this [guide to markdown](https://www.markdownguide.org/cheat-sheet/) too